Sources/SwiftSyntaxBuilder/gyb_generated/DeclBuildables.swift

```diff
@@ -0,0 +1,1082 @@
+    )
+  }
+}
+
+extension InitializerDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    initKeyword: TokenSyntax = Tokens.`init`,
+    optionalMark: TokenSyntax? = nil,
+    genericParameterClause: GenericParameterClause? = nil,
+    parameters: ParameterClause,
+    throwsOrRethrowsKeyword: TokenSyntax? = nil,
+    genericWhereClause: GenericWhereClause? = nil,
+    body: CodeBlock? = nil
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      initKeyword: initKeyword,
+      optionalMark: optionalMark,
+      genericParameterClause: genericParameterClause,
+      parameters: parameters,
+      throwsOrRethrowsKeyword: throwsOrRethrowsKeyword,
+      genericWhereClause: genericWhereClause,
+      body: body
+    )
+  }
+}
+
+extension DeinitializerDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    deinitKeyword: TokenSyntax = Tokens.`deinit`,
+    body: CodeBlock
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      deinitKeyword: deinitKeyword,
+      body: body
+    )
+  }
+}
+
+extension SubscriptDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    subscriptKeyword: TokenSyntax = Tokens.`subscript`,
+    genericParameterClause: GenericParameterClause? = nil,
+    indices: ParameterClause,
+    result: ReturnClause,
+    genericWhereClause: GenericWhereClause? = nil,
+    accessor: SyntaxBuildable? = nil
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      subscriptKeyword: subscriptKeyword,
+      genericParameterClause: genericParameterClause,
+      indices: indices,
+      result: result,
+      genericWhereClause: genericWhereClause,
+      accessor: accessor
+    )
+  }
+}
+
+extension ImportDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    importTok: TokenSyntax = Tokens.`import`,
+    importKind: TokenSyntax? = nil,
+    @AccessPathBuilder pathBuilder: () -> AccessPath = { .empty }
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      importTok: importTok,
+      importKind: importKind,
+      path: pathBuilder()
+    )
+  }
+}
+
+extension AccessorDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    modifier: DeclModifier? = nil,
+    accessorKind: TokenSyntax,
+    parameter: AccessorParameter? = nil,
+    asyncKeyword: String?,
```

Comment:
I have seen multiple placeses where the `async` keyword is a string.
Is this meant to be this way?

Reply:
Yes, that’s correct. `async` is a contextual keyword (which means that you can use it in identifiers etc.). I think that’s mostly for historical reasons. If I remember correctly, someone wanted to change it but it’s not a top priority.